tests/licensedcode/test_plugin_license_detection.py

```diff
@@ -126,6 +126,24 @@ def test_license_match_extra_words_3_seq():
         test_dir,
     ]
     run_scan_click(args)
+    test_loc = test_env.get_test_loc('plugin_license/extra-words/scan-extra-words-3-seq-with-copyrights-license.expected.json')
+    check_json_scan(test_loc, result_file, regen=REGEN_TEST_FIXTURES)
+
+
+def test_license_match_extra_words_3_seq_with_copyrights():
+    test_dir = test_env.get_test_loc('plugin_license/extra-words/scan-extra-words-3-seq-license-with-copyright/')
+    result_file = test_env.get_temp_file('json')
+    args = [
+        '--license',
```

Comment:
Can you also use the `--copyright` option here so we have copyright detection in the resource, we have to use this to resolve the issue.